runtime/src/accounts_db.rs

```diff
@@ -413,6 +416,8 @@ pub struct AccountsDB {
     dead_slots: RwLock<HashSet<Slot>>,
 
     stats: AccountsStats,
+
+    operating_mode: Option<OperatingMode>,
```

Comment:
Why option?

Reply:
It was to avoid users accidentally using/initializing an `AccountsDb::default()` without initializing the `operating_mode`. The alternative was to get rid of the `Defafult()` implementation and only allow creation through our constructors

Reply:
ok